docs-guides/source/model-prediction.md

```diff
@@ -160,3 +160,140 @@ img_as_np_array = np.reshape(img_as_np_array, model_expected_input_shape)
+```
+
+### Why Use a Compiled Model?
+
+When you initialize a model using (in Python) `model ct.models.MLModel("model.mlpackge")`, the Core ML Framework is invoked and the following steps occur, as shown in the following diagram.
+
+```{figure} images/model-lifecycle.png
+:alt: Initialize MLModel
+:align: center
+:class: imgnoborder
+
+This diagram is from [Improve Core ML integration with async prediction](https://developer.apple.com/videos/play/wwdc2023/10049/),
+presented at the Apple 2023 World Wide Developer Conference.
+```
+
+1. The `mlpackage` is [compiled](https://developer.apple.com/documentation/coreml/mlmodel/3929553-compilemodelaturl) into a file with extension `mlmodelc` . This step is usually very fast.
+
+2. The compiled model is then [instantiated](https://developer.apple.com/documentation/coreml/mlmodel/3022229-modelwithcontentsofurl) using the specified `compute_units`, and captured in the MLModelConfiguration config. 
+
+3. During instantiation, another compilation occurs for backend device specialization, such as for the Neural Engine (NE), which may take a few seconds or even minutes for large models. 
+    
+This device specialization step creates the final compiled asset ready to be run. This final compiled model is cached so that the expensive device optimization process does not need to run again. The cache entry is linked to the full file system path of the `mlmodelc` folder.
+
+As you create an MLModel object in Python using an `mlpackage`, it uses a temporary directory in a new location to place the `mlmodelc` folder. The `mlmodelc` file is then deleted after you have made predictions and the Python process has ended. 
+
+The next time you start a new Python process and create an MLModel, the compilation to `mlmodelc` and the subsequent device specialization occurs again. The cached set can’t be used again, because the location of `mlmodelc` has changed. 
+
+By storing the `mlmodelc` file to a fixed location first, and then initializing the MLModel from that location, you can make sure that the cache model generated remains active for subsequent loads, thereby making them faster. Let's see how you would do that in code. 
+
+### Predict From the Compiled Model 
+
+To use a compiled model file, follow these steps:
+
+1. Load a saved MLModel, or convert a model from a training framework (such as TensorFlow or PyTorch). 
+    
+    For instructions on converting a model, see [Load and Convert Model Workflow](load-and-convert-model). This example uses the [regnet_y_128fg](https://pytorch.org/vision/main/models/generated/torchvision.models.regnet_y_128gf.html) torchvision model and assumes that you have already converted it to a Core ML `mlpackage`.
+
+2. Get the compiled model directory by calling its `get_compiled_model_path` method. 
+    
+    For example, the following code snippet loads a saved MLModel (`"regnet_y_128gf.mlpackage"`) and gets the compiled path:
+    
+	```
+	mlmodel.save("regnet_y_128gf.mlpackage")
```

Comment:
This is not the code to load a saved model. This should be instead `mlmodel = ct.models.MLModel("regnet_y_128gf.mlpackage")`

Reply:
OK fixed, thanks!